sdks/python/apache_beam/transforms/external.py

```diff
@@ -104,6 +105,43 @@ def payload(self):
     """
     return self.build().SerializeToString()
 
+  def _get_schema_proto_and_payload(self, ignored_arg_format, *args, **kwargs):
```

Comment:
The `ignored_arg_format` param feels a bit awkward, and it'd be easy to accidentally forget it and grab the first arg as well (and I'm not sure the JavaClassLookupPayloadBuilder.IGNORED_ARG_FORMAT logic should be in this superclass as well). 

Instead, could we have the java-class calling one just make a dict using JavaClassLookupPayloadBuilder.IGNORED_ARG_FORMAT for the positional args, and then call shared code to go from a dict to a schema and payload?